src/pkg/packager/publish.go

```diff
@@ -112,7 +112,7 @@ func (p *Packager) Publish() (err error) {
 		return err
 	}
 
-	remote, err := oci.NewOrasRemote(ref)
+	remote, err := oci.NewOrasRemote(ref, ocispec.Platform{})
```

Comment:
we should have arch info here

Reply:
updated, now passing ing `oci.PlatformForArch(config.GetArch())`

Reply:
looks like this one was missed.

Reply:
whoops! Thanks again for the review!